src/main/java/org/assertj/core/api/AbstractByteAssert.java

```diff
@@ -375,7 +376,43 @@ public S isCloseTo(Byte expected, Offset<Byte> offset) {
     bytes.assertIsCloseTo(info, actual, expected, offset);
     return myself;
   }
-  
+
+  /** {@inheritDoc} */
```

Comment:
I have oriented myself on AbstractDoubleAssert which is full inheritDoc but ok, i will change it.